README.md

```diff
@@ -6,8 +6,63 @@ If you have any feedback or are interested in what we are doing please [contact
 
 API for managing and accessing Ably Spaces.
 
+**Class definitions are available [here](/docs/class-definitions.md).**
+
 ## Spaces
 
 A Space is an Ably concept that allows for shared realtime digital experiences such as chat, reactions and collaborative editing.
 
 A Space consists of users and storage with persistent presence.
```

Comment:
What do we mean by storage and persistent presence?

Reply:
@MymmiJ wrote that line, it's not part of this PR

Reply:
We should probably delete those, they are I think taken from our design-driven docs and don't describe what's available, but what we'd like to enable in the future. I imagine the conceptual documentation will eventually cover this.